src/main/java/mate/jdbc/Main.java

```diff
@@ -1,7 +1,60 @@
 package mate.jdbc;
 
+import java.util.ArrayList;
+import java.util.List;
+import mate.jdbc.lib.Injector;
+import mate.jdbc.model.Car;
+import mate.jdbc.model.Driver;
+import mate.jdbc.model.Manufacturer;
+import mate.jdbc.service.CarService;
+import mate.jdbc.service.DriverService;
+import mate.jdbc.service.ManufacturerService;
+
 public class Main {
+    private static final Injector injector = Injector.getInstance("mate.jdbc");
+    private static final ManufacturerService manufacturerService =
+            (ManufacturerService) injector.getInstance(ManufacturerService.class);
+    private static final CarService carService =
+            (CarService) injector.getInstance(CarService.class);
+    private static final DriverService driverService =
+            (DriverService) injector.getInstance(DriverService.class);
+
     public static void main(String[] args) {
-        // test your code here
+        Driver driverBob;
+        Driver driverJohn;
+        driverBob = driverService.create(new Driver("Bob", "124512"));
+        driverJohn = driverService.create(new Driver("John", "925031"));
+
+        manufacturerService.getAll().forEach(System.out::println);
+        driverService.getAll().forEach(System.out::println);
+
+        Manufacturer skoda;
+        Manufacturer fiat;
+        fiat = manufacturerService.create(new Manufacturer("Fiat", "France"));
+        skoda = manufacturerService.create(new Manufacturer("Skoda", "Czech"));
+        skoda.setName("WAG");
```

Comment:
```suggestion
        Manufacturer skoda = manufacturerService.create(new Manufacturer("Skoda", "Czech"));
        Manufacturer fiat = manufacturerService.create(new Manufacturer("Fiat", "France"));
        skoda.setName("WAG");
```
And why to set name one more time?